http2/src/test/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandlerTest.java

```diff
@@ -24,183 +25,202 @@
 
 public class HTTP2StreamHandlerTest {
 
-    private HTTP2StreamHandler http2StreamHandler;
-    private HTTP2Connection http2Connection;
-    private URL url;
-    private HTTP2SampleResult http2SampleResult;
-    private Stream stream;
-    private PushPromiseFrame pushPromisFrame;
-    private HeadersFrame headersFrame;
-    private MetaData.Response responseMetadata;
-    private Callback callback;
-    private DataFrame dataFrame;
+  private HTTP2StreamHandler http2StreamHandler;
+  private HTTP2Connection http2Connection;
+  private URL url;
+  private HTTP2SampleResult http2SampleResult;
+  private Stream stream;
+  private PushPromiseFrame pushPromisFrame;
+  private HeadersFrame headersFrame;
+  private MetaData.Response responseMetadata;
+  private Callback callback;
+  private DataFrame dataFrame;
+  private ResetFrame resetFrame;
```

Comment:
Since this field is only used in one method prefer using (and initializing) in that method (onResetTest) to ease readability and keep variables near to their usage.